tests/embed/test_video_i3d_r3d.py

```diff
@@ -0,0 +1,74 @@
+"""Reference: https://github.com/pytorch/vision/blob/master/test/test_models.py#L257"""
+
+import pytest
+import torch
+
+from kale.embed.video_i3d import InceptionI3d
+from kale.embed.video_res3d import BasicBlock, BasicStem, Bottleneck, Conv3DSimple, VideoResNet
+from kale.embed.video_se_i3d import SEInceptionI3DFlow, SEInceptionI3DRGB
+from kale.embed.video_se_res3d import se_r3d_18_flow, se_r3d_18_rgb
+from kale.utils.seed import set_seed
+
+# the default input shape is batch_size * num_channel * frame_per_segment * height * weight
+INPUT_BATCH_RGB = torch.randn(2, 3, 16, 112, 112)
+INPUT_BATCH_FLOW = torch.randn(2, 2, 8, 112, 112)
+SE_LAYERS = ["SELayerC", "SELayerT", "SELayerCoC", "SELayerMC", "SELayerMAC", "SELayerCT", "SELayerTC"]
+set_seed(36)
```

Comment:
I know that this won't fail the tests but it should be better to always set seed before using random functions, i.e., putting it before line 13 will be better, isn't it?